en/gallery.html

```diff
@@ -17,6 +17,7 @@
   <link rel="stylesheet" type="text/css" href="/css/kickstart.css" media="all" />                  <!-- KICKSTART -->
   <link rel="stylesheet" type="text/css" href="/style2.css" media="all" />
   <script type="text/javascript">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">

```

Comment:
in the future we may want to have responsive web, depending on the size of the device